src/h5web/dataset-visualizer/DatasetVisualizer.tsx

```diff
@@ -1,20 +1,20 @@
 import React, { ReactElement, useEffect, useContext } from 'react';
 import { useSetState, useTimeoutFn } from 'react-use';
 import type { HDF5Dataset, HDF5Value } from '../providers/models';
-import styles from './DatasetVisualizer.module.css';
+import styles from '../Visualizer.module.css';
 import VisSelector from './VisSelector';
 import VisDisplay from './VisDisplay';
-import type { Vis, DimensionMapping } from './models';
+import { DimensionMapping, Vis } from './models';
 import { VIS_DEFS } from '../visualizations';
 import { ProviderContext } from '../providers/context';
 import { useSupportedVis } from './hooks';
 
 interface Props {
-  dataset?: HDF5Dataset;
+  dataset: HDF5Dataset;
 }
 
 interface State {
-  activeVis?: Vis;
+  activeVis: Vis;
```

Comment:
I initialise the local state with a default `Vis`, which allows me to remove another bunch of checks.